samples/dnn/optical_flow.py

```diff
@@ -62,9 +64,9 @@ def motion_to_color(self, flow):
     parser = argparse.ArgumentParser(description='Use this script to calculate optical flow using FlowNetv2',
                                      formatter_class=argparse.ArgumentDefaultsHelpFormatter)
     parser.add_argument('-input', '-i', required=True, help='Path to input video file. Skip this argument to capture frames from a camera.')
-    parser.add_argument('--height', default=320, help='Input height')
-    parser.add_argument('--width',  default=448, help='Input width')
-    parser.add_argument('--proto', '-p', default='FlowNet2_deploy.prototxt', help='Path to prototxt.')
+    parser.add_argument('--height', default=320, type=int, help='Input height')
+    parser.add_argument('--width', default=448, type=int, help='Input width')
+    parser.add_argument('--proto', '-p', default='FlowNet2_deploy_anysize.prototxt', help='Path to prototxt.')
```

Comment:
Why do we need to change the default proto name?

Reply:
Sure we can keep the default proto name. This name (with `_anysize`) is the one I attached in the google drive.

Reply:
In `flownet2-models.tar.gz` from https://github.com/lmb-freiburg/flownet2/blob/master/models/download-models.sh default name is `FlowNet2_deploy.prototxt.template`

Reply:
Yes. However, the default `FlowNet2_deploy.prototxt.template` doesn't work with opencv-4.4.0 directly (maybe lacking certain layer's support). I modify the proto in `https://drive.google.com/open?id=19bo6SWU2p8ZKvjXqMKiCPdK8mghwDy9b` to have similar $-quoted variables link  `FlowNet2_deploy.prototxt.template`.